demos/classification_demo/python/README.md

```diff
@@ -130,60 +130,82 @@ and `<omz_dir>/data/dataset_classes/imagenet_2012.txt` labels file with all othe
+  --labels LABELS       Optional. Labels mapping file.
+  -topk {1,2,3,4,5,6,7,8,9,10}
+                        Optional. Number of top results. Default value is 5.
+                        Must be from 1 to 10.
 
 Inference options:
   -nireq NUM_INFER_REQUESTS, --num_infer_requests NUM_INFER_REQUESTS
                         Optional. Number of infer requests
   -nstreams NUM_STREAMS, --num_streams NUM_STREAMS
-                        Optional. Number of streams to use for inference on the CPU or/and GPU in throughput mode (for
-                        HETERO and MULTI device cases use format <device1>:<nstreams1>,<device2>:<nstreams2> or just
+                        Optional. Number of streams to use for inference on
+                        the CPU or/and GPU in throughput mode (for HETERO and
+                        MULTI device cases use format
+                        <device1>:<nstreams1>,<device2>:<nstreams2> or just
                         <nstreams>).
   -nthreads NUM_THREADS, --num_threads NUM_THREADS
-                        Optional. Number of threads to use for inference on CPU (including HETERO cases).
+                        Optional. Number of threads to use for inference on
+                        CPU (including HETERO cases).
 
 Input/output options:
   --loop                Optional. Enable reading the input in a loop.
   -o OUTPUT, --output OUTPUT
                         Optional. Name of the output file(s) to save.
   -limit OUTPUT_LIMIT, --output_limit OUTPUT_LIMIT
-                        Optional. Number of frames to store in output. If 0 is set, all frames are stored.
+                        Optional. Number of frames to store in output. If 0 is
+                        set, all frames are stored.
   --no_show             Optional. Don't show output.
   --output_resolution OUTPUT_RESOLUTION
-                        Optional. Specify the maximum output window resolution in (width x height) format. Example:
-                        1280x720. Input frame size used by default.
+                        Optional. Specify the maximum output window resolution
+                        in (width x height) format. Example: 1280x720. Input
+                        frame size used by default.
   -u UTILIZATION_MONITORS, --utilization_monitors UTILIZATION_MONITORS
                         Optional. List of monitors to show initially.
 
 Input transform options:
   --reverse_input_channels
-                        Optional. Switch the input channels order from BGR to RGB.
+                        Optional. Switch the input channels order from BGR to
+                        RGB.
   --mean_values MEAN_VALUES MEAN_VALUES MEAN_VALUES
-                        Optional. Normalize input by subtracting the mean values per channel. Example: 255 255 255
+                        Optional. Normalize input by subtracting the mean
+                        values per channel. Example: 255 255 255
```

Comment:
```suggestion
                        values per channel. Example: 255.0 255.0 255.0
```

Reply:
You should update https://github.com/openvinotoolkit/open_model_zoo/blob/e71b236ca0cdaa03b09f1352e61aa04e8c41188d/demos/classification_demo/python/classification_demo.py#L95 to have `.0` in first place, and regenerate README after that.

Reply:
Done.